packages/components/src/schemas/main.ts

```diff
@@ -35,3 +43,18 @@ export const getComponentSchema = (
     ...definitions,
   }
 }
+
+const LAYOUT_METADATA_MAP = {
```

Comment:
no actionable needed - just voicing my thoughts out loud. wdyt of having a similar API to `JSONForms` here in that we just export `tester + render` and our component in `RootStateDrawer` just uses that to render?